src-vue/src/views/PlayView.vue

```diff
@@ -32,6 +38,13 @@ export default defineComponent({
             <div v-if="northstarVersion !== ''" class="fc_changelog__link" @click="showChangelogPage">
                 (see patch notes)
             </div>
+            <div v-if="playerCount >= 0" class="fc-stats__container">
+                {{ playerCount }} Players,
```

Comment:
I think we should not use uppercase here

Reply:
Changed to lowercase in ea1efa939681e79e4b2ea513bf683628adc63d14